provider/fs/src/export/fs_exporter.rs

```diff
@@ -2,13 +2,17 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+// We're using SipHash, which is deprecated, but we want a stable hasher
+// (we're fine with it not being cryptographically secure since we're just using it to track diffs)
+#![allow(deprecated)]
```

Comment:
Nit: Can you make the annotation apply to only individual lines of code? It's good to get deprecation warnings in general for other things.